scripts/dmc_reset.py

```diff
@@ -262,7 +261,7 @@ def main():
         logger.info("No hardware map provided, using first ST-Link")
```

Comment:
I think we can drop this `if` block with these lines, since jtag_id should already be set to `None` by default.

Reply:
Fixed!